build.xml

```diff
@@ -182,52 +182,44 @@
     <!-- Create demo users. -->
     <target name="create-demo-users" description="Create demo users.">
         <foreach list="${drupal.demo.users}" param="drupal.demo.user" target="create-demo-user" delimiter="," />
-        <!-- Create a user with only "authenticated user" role assigned. -->
+        <foreach list="${drupal.demo.authenticated_users}" param="drupal.demo.user" target="create-demo-authenticated_user" delimiter="," />
+    </target>
+
+    <!-- Create a demo user. Subtarget of "create-demo-users". -->
+    <target name="create-demo-user">
```

Comment:
That's not true, these users are created in the install hook of the demo content module. Let's get rid of these changes. There is no reason to make more than one demo user for the same role.